node/libs/roles/src/proto/validator.proto

```diff
@@ -3,19 +3,21 @@ syntax = "proto3";
 package zksync.roles.validator;
 
 import "zksync/std.proto";
+import "zksync/roles/attester.proto";
 
 message Genesis {
   reserved 1,2;
   reserved "fork","validators";
   optional uint64 chain_id = 5; // required
   optional uint64 fork_number = 6; // required; ForkNumber
   optional uint64 first_block = 7; // required; BlockNumber
+  repeated attester.WeightedAttester attesters = 4;
```

Comment:
attesters are not supposed to be fixed per genesis, but rather might change each epoch, so move this field to the section below